tmt/utils.py

```diff
@@ -4789,3 +4786,15 @@ def is_selinux_supported() -> bool:
     """
     with open('/proc/filesystems', 'r') as file:
         return any('selinuxfs' in line for line in file)
+
+
+def get_workdir_root() -> str:
+    """
+    The 'workdir_root' variable is initialized from TMT_WORKDIR_ROOT
+    environment variable, if present, otherwise defaults to WORKDIR_ROOT.
+    """
+    if 'TMT_WORKDIR_ROOT' in os.environ:
+        workdir_root = os.environ['TMT_WORKDIR_ROOT']
+    else:
+        workdir_root = str(WORKDIR_ROOT)
+    return workdir_root
```

Comment:
Definitely return `Path` - the value returned is not just any random string, it's a *path*, therefore a `Path` instance shall be produced.

Reply:
Updated.